Cargo.toml

```diff
@@ -19,11 +19,11 @@ path = "src/passthrough.rs"
 tokio = { version = "1.17.0", features = [ "full" ] }
 futures-util = { version = "0.3.21" }
 
-zeroconf = { version = "0.10.5", path = "../zeroconf-rs/zeroconf", optional = true }
-mdns = { version = "3.0.0", path = "../mdns" }
```

Comment:
This crate doesn't compile on the latest rustc last time I checked, hence the fork

https://github.com/jkcoxson/mdns.git

Reply:
This compiles as-is with rustc 1.70.0. Your commit doesn't change anything since "0.4" selects any version 0.4.* (cargo update in this project would have fixed that as well, since mdns doesn't have a committed lock file).